### ISSUE TITLE
Failed to convert the mask-rcnn keras model using MO

### ISSUE THREAD

**Issue by LingCoder1997** [openvinotoolkit/openvino#15984]
##### System information (version)

- OpenVINO=> :2022.3.0 (self built from github code master)
- Operating System / Platform => :Linux:
- Compiler => :VScode:
- Problem classification => :Model Conversion:
- Framework: TensorFlow-1.15.5 
- Model name: Mask-RCNN (https://github.com/matterport/Mask_RCNN)

##### Detailed description
Hi! I have tried to convert my Mask-RCNN model from keras to OpeVINO supported type. Before the MO convertion, I made some preperation which includes converting the keras model into both ONNX and tensorflow. However, when I applied the MO convertion from ONNX to IR, the command is like below:

_mo --input_model my_MR_model.onnx --output_model IR_model --data_type FP16 --static_shape --use_new_frontend --batch 1_

Then I received the error log below:

[ERROR] Check '(false)' failed at src/frontends/onnx/frontend/src/op/resize.cpp:98
while validating ONNX node '<None(Resize)': Resize__1367>
tf_crop_and_resize - this type of coordinate transformation mode is not supported. Choose one of the following modes: tf_half_pixel_for_nn, asymmetric, align_corners, pytorch_half_pixels, half_pixel

After checking the keras model-build function, I found the author used the op: tf.image.crop_and_resize() (From [here](https://github.com/matterport/Mask_RCNN/blob/3deaec5d902d16e1daf56b62d5971d428dc920bc/mrcnn/model.py#L421))to do the ROI alignment. I supposed this might be source of the problem.

Then I also tried to convert the model from tensorflow(.pb) to IR. Using the command below:
_mo --input_model my_MR_model.pb --output_model IR_model --data_type FP16 --static_shape --batch 1_

However, a new error came out:

[ ERROR ] Check "is_converted" failed at src/frontends/tensorflow/src/translate_session.cpp:219:
FrontEND API failed with OpConversionFailure: :
[TensorFlow Frontend] Internal error: No translator found fro DenseToDenseSetOperation node.

I  am not quite sure which op, from original Keras model, generated this op when converting from Keras to tensorflow. 

So I wonder if it is possible to solve either problem above that I can convert my model to IR framework. Also it is also acceptable to replace the specific op from the original Keras model by other ops.
##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [ ] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [ ] There is reproducer code and related data files: images, videos, models, etc.

**Comment by rkazants** [comment]
Hi @LingCoder1997,

I checked if we can provide transformation for this operation `DenseToDenseSetOperation`. So it will be difficult to decompose it using OpenVINO opset due to several reasons: input format is dense and output format is sparse; difficult to perform `set_operation` using the current opset and due to dense format.

It seems to me that we should think of extension of our OpenVINO opset. @LingCoder1997, may be to you can share the model (with fake weights it is fine) and I will take a look other workarounds for this operation?

@andrei-kochin, @tomdol, @mlukasze, [`DenseToDenseSetOperation`](https://www.tensorflow.org/jvm/api_docs/java/org/tensorflow/op/sparse/DenseToDenseSetOperation) is potential candidate to our opset.

Best regards,
Roman

**Comment by akladiev** [comment]
This issue will be closed in 2 weeks in case of no activity.

**Comment by avitial** [comment]
Ref. 111484

**Comment by rkazants** [comment]
@LingCoder1997, can you please try to convert model with the latest OpenVINO package: https://github.com/openvinotoolkit/openvino/releases/tag/2023.0.0.dev20230427

I would like to see if any other unsupported operations are met for this model

Best regards,
Roman

**Comment by andrei-kochin** [comment]
Closed due to no activity
